docs/network-flow-visibility.md

```diff
@@ -283,7 +286,44 @@ it is deployed following the [deployment steps](#deployment-steps-1), the
 ClickHouse server is already exposed via a K8s Service, and no further
 configuration is required. If a different FQDN or IP is desired, please use
 the URL for `clickHouse.databaseURL` in the following format:
-`tcp://<ClickHouse server FQDN or IP>:<ClickHouse TCP port>`.
+`<Protocol>://<ClickHouse server FQDN or IP>:<ClickHouse port>`.
+
+#### Support secure connections to ClickHouse database
+
+Since Antrea v1.13, you can enable TLS when connecting to ClickHouse Server by
+setting `clickHouse.databaseURL` with protocol `tls` or `https`. You can also
+change the value of `clickHouse.tls.insecureSkipVerify` to determine whether
+to skip the verification of the server's certificate. If you want to provide
+a custom CA certificate, you can set `clickHouse.tls.caCert` to `true` and
+the flow Aggregator will read the certificate key pair from the`clickhouse-ca`
+Secret.
+
+Make sure to follow the following form when creating the `clickhouse-ca` Secret
+with the custom CA certificate:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: clickhouse-ca
+  namespace: flow-aggregator
+data:
+  ca.crt: <BASE64 ENCODED CA CERTIFICATE>
+```
+
+You can use `kubectl apply -f <PATH TO SECRET YAML>` to create the above secret
+, or use `kubectl create secret`:
+
+```bash
+kubectl create secret generic clickhouse-ca -n flow-aggregator --from-file=ca.crt=<PATH TO CA CERTIFICATE>
+```
+
+#### Configuration for ClickHouse pre Antrea v1.13
+
+We don't support secure connection to ClickHouse database prior to Antrea v1.13.
+We only support TCP when connecting to ClickHouse server from Flow Aggregator.
```

Comment:
I am still not sure I understand the value of this section, or why it is its own section

I feel like it should just be a quick note at the end of the previous section:

> Prior to Antrea v1.13, secure connections to ClickHouse are not supported, and TCP is the only supported protocol when connecting to the ClickHouse server from the Flow Aggregator.

Reply:
The reason I added this section is because I saw we usually add extra section when we have configuration changes in other document, like node-port-local. 
I've updated it. Thanks.